### DISCUSSION TITLE
Is placing code under GPL 2 allowed in private repositories?

### DISCUSSION THREAD

**Discussion by Alex-Pismenny** [community/community#48729] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Is placing code under GPL version 2 of the GNU General Public License as published by the Free Software Foundation legal in a private repository?

We do not intend to redistribute those.

**Comment by airtower-luna** [comment]
*Having* the code as such most likely is (if you haven't done anything that'd invalidate the license), but the question is what you then *do* with the code. The GPL defines the conditions under which you'd have to share the code, you'll have to check those against what you're doing.

And the usual warning: If you need legal advice, ask an actual lawyer. :wink:

**Reply by Alex-Pismenny** [reply]
We use it read-only. It's a build system for an on-board Unix. We compile our own code using it.

Thank you.
